official/guides/child_account_update/export.py

```diff
@@ -0,0 +1,154 @@
+# This script will export USPS credentials for all child accounts to a CSV file
+
+# NOTE: Due to the nature of the easypost API library,
+# this script will run synchronously and therefore may take a while to complete if you have a lot of child accounts
+
+from typing import List, Dict, Union
+
+import easypost
+import argparse
+import csv
+
+OUTPUT_FILE_NAME = "child_accounts.csv"
+
+parser = argparse.ArgumentParser(description="Export all child account USPS production credentials")
+parser.add_argument("-k", "--key", required=True, help="Parent production API key")
+
+
+def authenticate(key: str):
+    """
+    Authenticate with the given API key
+
+    :param key: API key to authenticate with
+    :type key: str
+    :raises ValueError: If the key is invalid
+    :return: None
+    """
+    if not key:
+        raise ValueError("API key not provided")
+    easypost.api_key = key
+
+
+def get_production_key(keys: List) -> Union[str, None]:
+    """
+    Get the production key from a list of keys
+
+    :param keys: List of keys
+    :type keys: List
+    :return: Production key if found, None otherwise
+    :rtype: str | None
+    """
+    for key in keys:
+        if key["mode"] == "production":
+            return key["key"]
+    return None
+
+
+def process_child(child: easypost.User) -> Dict:
+    """
+    Process a child account
+
+    :param child: Child account object
+    :type child: easypost.User
+    :return: Dictionary of child account information
+    :rtype: Dict
+    """
+    child_prod_key = get_production_key(child.keys)
+    authenticate(child_prod_key)
+    accounts = get_usps_accounts()
+    credentials = []
+    for account in accounts:
+        account_details = {
+            "test": account.test_credentials,
+            "prod": account.credentials,
+        }
+        if account_details.get("test"):
+            details = {**account_details.get("test"), "carrier_account_id": account.id, "credential_type": "test"}
+            credentials.append(details)
+        if account_details.get("prod"):
+            details = {**account_details.get("prod"), "carrier_account_id": account.id, "credential_type": "prod"}
+            credentials.append(details)
+
+    return {"id": child.id, "accounts": credentials}
```

Comment:
Clarity is always good

```suggestion
    return {"id": child.id, "usps_accounts": credentials}
```

Reply:
Resolved